sig-architecture/charter.md

```diff
@@ -1,40 +1,175 @@
+    <td>What is (and is not) Kubernetes</td>
+  </tr>
+  <tr>
+    <td>Conformance definition</td>
+    <td>Reviewing, approving, and driving changes to the conformance test suite; reviewing, guiding, and creating new conformance profiles</td>
+    <td>Test list, Test guidelines</td>
+  </tr>
+  <tr>
+    <td>Architectural governance</td>
+    <td>Establishing and documenting design principles, documenting and evolving the system architecture, reviewing, curating, and documenting new extension patterns</td>
+    <td>Kubernetes Design and Architecture, Design principles</td>
+  </tr>
+  <tr>
+    <td>API governance</td>
+    <td>Establishing and documenting conventions for system and user-facing APIs, define and operate the APl review process, final API implementation consistency validation, co-own top-level API directories with API machinery</td>
+    <td>API conventions, API linter</td>
+  </tr>
+  <tr>
+    <td>Deprecation policy</td>
+    <td>Maintaining, evolving, and enforcing the deprecation policy</td>
+    <td>Deprecation policy</td>
+  </tr>
+  <tr>
+    <td>Code organization</td>
+    <td>Overall code organization, including github repositories and branching methodology, top-level and pkg OWNERS of kubernetes/kubernetes, kubernetes-template-project</td>
+    <td>kubernetes-template-project</td>
+  </tr>
+  <tr>
+    <td>KEP</td>
+    <td>Develop and drive technical enhancement review process</td>
+    <td>Process, Template</td>
+  </tr>
+</table>
+
+
+## Roles
+
+The following roles are required for the SIG to properly function. In the event that any role is unfilled, the SIG will make a best effort to fill it, and any decisions reliant on a missing role will be postponed until the role is filled.
+
+* **Chair**
+
+    * Number: 2-3
+
+    * Run operations and processes governing the SIG
+
+    * An initial set of chairs was established at the time the SIG was founded.
+
+    * Chairs *MAY* decide to step down at anytime and propose a replacement, who must be approved by all of the other chairs. This *SHOULD* be supported by a majority of SIG Members.
+
+    * Chairs *MAY* select additional chairs by consensus. This *SHOULD* be supported by a majority of SIG Members.
+
+    * Chairs *MUST* remain active in the role and are automatically removed from the position if they are unresponsive for > 3 months and *MAY* be removed by consensus of the other Chairs and all of the Technical leads if not proactively working with other Chairs to fulfill responsibilities.
+
+    * A majority of chairs cannot be from a single company. If a chair’s change of company affiliation causes this threshold to be exceeded, one of the chairs must step down and propose a replacement, as per the procedure above.
```

Comment:
I think we should say "you can not have more than one chair from any one company"

Given 2 or 3 chairs that's implicitly true anyway.

Reply:
I'm fine with that.  "A majority of chairs" was my wording, and intended to be more future-proof, and resilient to changes in the number of chairs.